README.md

```diff
@@ -1,119 +1,84 @@
-- add a stanza for the plugin to the config
-    - vi kbase-ui/dev/config/ui/dev/build.yml
-    - in the plugins section add something like the following. 
-        - You will need to set ```root``` to the directory containing the plugin repo.
+- OPTION A, use your local directory, don't use make build after each edit
+*Important* Running ```make build``` again will destroy this link.
+```
+rm -rf ../build/build/client/modules/plugins/data-landing-pages
+ln -s ../kbase-ui-plugin-data-landing-pages/src/plugin ../build/build/client/modules/plugins/data-landing-pages
+```
 
+- OPTION B, use build.yml and make build after each edit
+  - Edit the config dev/config/ui/dev/build.yml, and modify the stanza for kbase-ui-plugin-data-landing-pages.
+    *Note* that this directory must be the parent of kbase-ui-plugin-data-landing-pages.
 ```
     -
         name: data-landing-pages
         globalName: kbase-ui-plugin-data-landing-pages
-        version: 0.1.0
+        version: x.y.z
         cwd: src/plugin
         source:
-            # uncomment the bower property to have the build fetch from bower/github
-            # bower: {}
-            # use something like this to feed bower a specific alternate version spec.
-            # in this case a specific github repo and branch
-            # bower:
-            #     version: you/kbase-ui-plugin-data-landing-pages#master
-            # comment this out if bower us activated
-            directory: 
-                # NB the root is the directory which contains this plugin, which
-                # must be located within a directory whose name is the globalName
-                root: /Users/you/work/kbase/sprints/data-api-widgets
-``` 
-    
-This setup will allow the "classic" edit-build-refresh workflow to integrate the plugin. Since the data api landing page repo contains just data widgets, there is no need for further configuration of the ui.
-
-If you hop back into kbase-ui and issue ```make build``` the plugin will be operational.
-
-You can test this by visiting a dataview url which should invoke an appropriate object (see the data api demo menu item which provides links to inspect compatible objects -- the references can pasted into the dataview url.)
-
-For example, one available in CI now is: http://localhost:8080/#dataview/1779/1006539/1
-
-### Faster Develop Workflow
-
-It is very simple to set up a faster edit-refresh workflow. Simply link your plugin directly into the build and refrain from running *make build*. For example, something like this from the parent directory of kbase-ui and the plugin:
+            #bower: {}
+            directory:
+                root: /absolute/path/to/parent/
+```
 
+4. Start kbase-ui
+This starts a static nodejs server at :8080.
 ```
-rm -rf kbase-ui/build/build/client/modules/plugins/data-landing-pages
-ln -s `pwd`/kbase-ui-plugin-data-landing-pages/src/plugin kbase-ui/build/build/client/modules/plugins/data-landing-pages
+make start
```

Comment:
need to chdir back up from 'dev' that you were in for the rm & ln commands (or make those not start with ../)